tensorflow/python/keras/utils/layer_utils_test.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for Keras Layer utils."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+
+from tensorflow.python import keras
+from tensorflow.python.keras.utils import layer_utils
+from tensorflow.python.platform import test
+
+
+class ConvertDataFormatTest(test.TestCase):
+
+  def test_convert_format(self):
+    def get_model(shape, data_format):
+      model = keras.Sequential()
+      model.add(
+          keras.layers.Conv2D(filters=2, kernel_size=(4, 3),
+                              input_shape=shape, data_format=data_format))
+      model.add(keras.layers.Flatten())
+      model.add(keras.layers.Dense(5))
+      return model
+
+    for data_format in ['channels_first', 'channels_last']:
```

Comment:
Use a test parameterization for cases like this, rather than an internal loop in the test. Unit tests should not use loops in general. This applies to a few other of your PRs as well.

Reply:
Sure, will modify according to review comments on this PR and other open PR also. Thanks.